modules/vpc-openshift/README.md

```diff
@@ -33,6 +33,7 @@ module "vpc_openshift_cluster" {
   kms_config                      = var.kms_config
```

Comment:
Update the vpc-openshift example (directory: examples/vpc-openshift) as well with patch_version ( showing how to pass patch_version from example template)